_includes/selector.html

```diff
@@ -688,44 +646,38 @@
             },
             disableUnsupportedCuda(cuda_version) {
                 var isDisabled = false;
-                if (this.active_img_options.includes("development") && cuda_version !== this.getNewestCudaVer()) {
-                    isDisabled = true;
-                }
                 if (this.active_additional_packages.includes("TensorFlow") && cuda_version !== "11.2") isDisabled = true;
-                if (this.active_os_ver === 'Ubuntu 22.04' && cuda_version !== '11.8' && this.active_method === 'Docker') isDisabled = true;
-                if (this.active_method !== 'pip' && cuda_version === '12.0') isDisabled = true;
+                if (this.active_arch === "arm" && cuda_version === "12.0") isDisabled = true;
 
                 return isDisabled;
             },
             disableUnsupportedPython(python_version) {
                 var isDisabled = false;
                 return isDisabled;
             },
-            disableUnsupportedImgOS(os) {
+            disableUnsupportedImgType(type) {
                 var isDisabled = false;
-                if (this.active_arch === "arm" && os === "CentOS 7") isDisabled = true;
-                if (this.active_cuda_ver !== '11.8' && os === 'Ubuntu 22.04' && this.active_method === 'Docker') isDisabled = true;
-                if (this.active_arch === "arm" && os === 'Ubuntu 20.04' && this.active_method === 'Docker') isDisabled = true;
+                if (this.active_arch === "arm" && type !== "core") isDisabled = true;
```

Comment:
This should be checking either `"Base"` or `"Notebooks"` and not `"core"`, right? I'm not entirely sure what this logic is meant to be doing -- do we support both base/notebooks with ARM?

Reply:
Hold-over from old logic, missed it when deleting old Docker-related logic.

Reply:
Resolved in https://github.com/rapidsai/docs/pull/414/commits/cb8e582dd6c332903e2cff27ed440967d7d0f222